ecosystem/sep-0024.md

```diff
@@ -170,26 +171,183 @@ There are several possible kinds of response, depending on whether the anchor ne
+5. From there the anchor will continue the [deposit flow](#basic-anchor-implementation)
+
+##### Anchor Feature: Extend Transaction history endpoints For Claimable Balances
+
+Wallets query the [transaction endpoint](#transaction-history) to see and display the status of deposits.
+Anchors should extend their transaction(s) endpoint response to include the `claimable_balance_id` that the anchor added to the [transaction field](####Fields-for-deposit-transactions).
+This will help wallets to provide UI to showcase that their user's deposit was successful and theres an [available claimable balance for them.](#wallet-feature-claimable-balance-polling)
+
+```
+GET TRANSFER_SERVER_SEP0024/transactions
+```
+
+```json
+{
+  "transactions": [
+    {
+      "id": "82fhs729f63dh0v4",
+      "kind": "deposit",
+      "status": "pending_external",
+      "status_eta": 3600,
+      "external_transaction_id": "2dd16cb409513026fbe7defc0c6f826c2d2c65c3da993f747d09bf7dafd31093",
+      "more_info_url": "https://youranchor.com/tx/242523523",
+      "amount_in": "18.34",
+      "amount_out": "18.24",
+      "amount_fee": "0.1",
+      "started_at": "2017-03-20T17:05:32Z",
+      "claimable_balance_id": null,
+    },
+    {
+      "id": "82fhs729f63dh0v4",
+      "kind": "deposit",
+      "status": "completed",
+      "amount_in": "500",
+      "amount_out": "495",
+      "amount_fee": "3",
+      "started_at": "2017-03-20T17:00:02Z",
+      "completed_at": "2017-03-20T17:09:58Z",
+      "more_info_url": "https://youranchor.com/tx/242523523",
+      "stellar_transaction_id": "17a670bc424ff5ce3b386dbfaae9990b66a2a37b4fbe51547e8794962a3f9e6a",
+      "external_transaction_id": "2dd16cb409513026fbe7defc0c6f826c2d2c65c3da993f747d09bf7dafd31093",
+      "claimable_balance_id": "00000000c2d8c89264288dbde8488364fd3fd30850fd4e7fbf6d1e9809702558afa4fdea",
+    }
+  ]
+}
+```
+
+```
+GET TRANSFER_SERVER_SEP0024/transaction
+```
+
+Example response:
+
+```json
+{
+  "transaction": {
+      "id": "82fhs729f63dh0v4",
+      "kind": "deposit",
+      "status": "completed",
+      "status_eta": 3600,
+      "external_transaction_id": "2dd16cb409513026fbe7defc0c6f826c2d2c65c3da993f747d09bf7dafd31093",
+      "more_info_url": "https://youranchor.com/tx/242523523",
+      "amount_in": "18.34",
+      "amount_out": "18.24",
+      "amount_fee": "0.1",
+      "started_at": "2017-03-20T17:05:32Z",
+      "claimable_balance_id": "00000000c2d8c89264288dbde8488364fd3fd30850fd4e7fbf6d1e9809702558afa4fdea",
+    }
+}
+```
+
+##### Anchor Claimable Balance Claimants and Predicates
+
+ Predicates are one of the [Claimable balance parameters](https://developers.stellar.org/docs/glossary/claimable-balance/#parameters) used when craftting a Claimable Balance transaction.
+They're used as conditions to meet in order for the recipient to claim their balance.
+      It's recommended to use the `UNCONDITIONAL` predicate to allow the claimant to claim the balance without any barriers.
```

Comment:
Remove the indent. I've noticed you like putting 1 sentence per line too, which is fine I guess but it may not be necessary. There's nothing wrong with multi-sentence lines.